plugin/controller/test/mcp/mcp.test.ts

```
@@ -93,7 +93,7 @@ describe('plugin/controller/test/mcp/mcp.test.ts', () => {
       return app.close();
     });
 
-    it('sse should work', async () => {
+    it.only('sse should work', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)